templates/error.html

```diff
@@ -0,0 +1,418 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sivustolla on häiriö</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        background-color: #f0f0f0;
+        text-align: center;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        display: flex;
```

Comment:
Header does not align with the other content on wide screens. It keeps on the left while footer and content are centered. Would look more consistent if the header aligns the same way.